### PR TITLE
5412 tm edit cycle cleanup

### PR DESCRIPTION
**Pull request by M00staff**
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/947)

tiny fixes testing picked up

### CODE REVIEW

File: src/Components/CycleManagement/AssignmentCycleEdit.jsx [hunk 1]
```diff
@@ -419,7 +419,7 @@ const AssignmentCycleEdit = ({ isAO, match }) => {
           <span className="bs-validation-container">
             <select
               id="cycleCategory"
-              defaultValue={cycleCategory.value}
```
Comment (M00staff) [inline src/Components/CycleManagement/AssignmentCycleEdit.jsx:422]: these values originally had both a label & value ... but I changed it to only store the value and didn't update the default value when you open the edit

File: src/Components/CycleManagement/NewAssignmentCycle.jsx [hunk 2]
```diff
@@ -164,8 +164,8 @@ const NewAssignmentCycle = (props) => {
               className="exclusivePosition-new"
               id="exclusivePosition-new"
               name="exclusivePosition-new"
-              checked={exclusivePosition}
-              onChange={() => setExclusivePosition(e => !e)}
```
Comment (M00staff) [inline src/Components/CycleManagement/NewAssignmentCycle.jsx:168]: backend PR should explains this

https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/947

Comment (jlam-mpc) [reply src/Components/CycleManagement/NewAssignmentCycle.jsx:168]: Matches what I expected looks good

File: src/Components/CycleManagement/NewAssignmentCycle.jsx [hunk 3]
```diff
@@ -176,8 +176,8 @@ const NewAssignmentCycle = (props) => {
               className="postViewable-new"
               id="postViewable-new"
               name="postViewable-new"
-              checked={postViewable}
-              onChange={() => setPostViewable(e => !e)}
+              value={postViewable === 'Y'}
+              onCheckBoxClick={e => setPostViewable(e ? 'Y' : 'N')}
```
Comment (jlam-mpc) [inline src/Components/CycleManagement/NewAssignmentCycle.jsx:180]: Looks good

File: src/Components/CycleManagement/NewAssignmentCycle.jsx [hunk 4]
```diff
@@ -15,8 +15,8 @@ const NewAssignmentCycle = (props) => {
   const [assignmentCycle, setAssignmentCycle] = useState('');
   const [cycleCategory, setCycleCategory] = useState('');
   const [cycleStatus, setCycleStatus] = useState('');
-  const [exclusivePosition, setExclusivePosition] = useState(false);
-  const [postViewable, setPostViewable] = useState(false);
+  const [exclusivePosition, setExclusivePosition] = useState('N');
+  const [postViewable, setPostViewable] = useState('N');
```
Comment (jlam-mpc) [inline src/Components/CycleManagement/NewAssignmentCycle.jsx:19]: Just went through codebase to make sure all instances of these two values have adapted to your backend changes and looks good

File: src/Components/CycleManagement/AssignmentCycleEdit.jsx (hunk 1 above)
Comment (jlam-mpc) [reply src/Components/CycleManagement/AssignmentCycleEdit.jsx:422]: Good catch

**Review by jlam-mpc — APPROVED** [review]

**Pull request merged**